src/Bindings/Browser/MultiFilingService.php

```diff
@@ -10,22 +10,22 @@
  * file that was distributed with this source code.
  */
 
+use Dkd\PhpCmis\Constants;
 use Dkd\PhpCmis\Data\ExtensionDataInterface;
 use Dkd\PhpCmis\MultiFilingServiceInterface;
 
 /**
  * MultiFiling Service Browser Binding client.
  */
-class MultiFilingService extends AbstractBrowserBindingService implements MultiFilingServiceInterface
-{
+class MultiFilingService extends AbstractBrowserBindingService implements MultiFilingServiceInterface {
     /**
      * Adds an existing fileable non-folder object to a folder.
      *
-     * @param string $repositoryId The identifier for the repository.
-     * @param string $objectId The identifier for the object.
-     * @param string $folderId The folder into which the object is to be filed.
-     * @param boolean $allVersions Add all versions of the object to the folder if the repository supports
-     *      version-specific filing. Defaults to <code>true</code>.
+     * @param string                      $repositoryId The identifier for the repository.
+     * @param string                      $objectId The identifier for the object.
+     * @param string                      $folderId The folder into which the object is to be filed.
+     * @param boolean                     $allVersions Add all versions of the object to the folder if the repository
+     *     supports version-specific filing. Defaults to <code>true</code>.
      * @param ExtensionDataInterface|null $extension
      */
```

Comment:
Should I add it even if it does not return anything (as the Java CMIS library does)?